apps/src/code-studio/pd/application/principalApproval/PrincipalApprovalApplication.jsx

```diff
@@ -46,10 +46,30 @@ const PrincipalApprovalApplication = props => {
   };
 
   const onSuccessfulSubmit = () => {
-    analyticsReporter.sendEvent(EVENTS.ADMIN_APPROVAL_RECEIVED_EVENT);
+    logSuccessfulApprovalSubmission();
     window.location.reload(true);
   };
 
+  const logSuccessfulApprovalSubmission = () => {
+    // Log admin approval received
+    analyticsReporter.sendEvent(EVENTS.ADMIN_APPROVAL_RECEIVED_EVENT);
+
+    // Log application status change to 'unreviewed'
+    const url = `/api/v1/pd/application/principal_approval/get_app_by_guid/${
+      props.teacherApplication.application_guid
+    }`;
+    $.ajax({
+      method: 'GET',
+      url: url,
+      dataType: 'json'
+    }).done(teacherApplication => {
```

Comment:
What should happen if this ajax request fails? Should the event still be sent, or no?

Reply:
Rather than making a separate ajax request here, thoughts about passing in the teacher application id on page load? I believe you can do that fairly easily by adding it in the controller here https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/pd/application/principal_approval_application_controller.rb#L19. Then you wouldn't need yet another ajax request to send this event

Reply:
Oh great point that's a much simpler solution! On it!

Reply:
Nice catch Meg! sorry I missed the page load option here when we talked about this Turner!

Reply:
I think I also missed it when we talked about it the first time!